gold-cc-cvc-input.html

```diff
@@ -63,8 +67,9 @@
         display: block;
       }
 
-      iron-icon {
+      .cvc-icon {
```

Comment:
I've just tested it with the mixin directly on iron-icon, and with this in the demo:
```
--gold-cc-cvc-input-icon: {
      padding-left: 40px;
      background: red;
    }
```

And it seems to work fine:
<img width="104" alt="screen shot 2016-04-15 at 1 56 17 pm" src="https://cloud.githubusercontent.com/assets/1369170/14574686/e980a5b2-0311-11e6-84bf-f5a1301961de.png">

Please give it another try?